app/controllers/passengers_controller.rb

```diff
@@ -0,0 +1,50 @@
+class PassengersController < ApplicationController
+  def index
+    @passengers = Passenger.all
```

Comment:
By doing this first, you'll be doing an extra database query that you don't need. You _only_ need the queries in the conditionals.